test/Microsoft.ML.Core.Tests/UnitTests/ColumnTypesVSTask.cs

```diff
@@ -0,0 +1,115 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.IO;
+using Microsoft.ML.Data;
+using Microsoft.ML.Transforms.Image;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Microsoft.ML.RunTests
+{
+    [TestClass]
+    public sealed class ColumnTypesVSTask
+    {
+        public ColumnTypesVSTask()
+        {
+        }
+
+        [TestMethod]
+        // [TestCategory("RunSpecificTest")]
```

Comment:
Is this line meant to be a comment?

Reply:
The machinelearning-testdata repository contains "binary data that is consumed by tests", I don't think that should contain the ProcDump.exe executable nor its end user licence agreement.
That is a demonstration of marking a test to be run specifically in our CI builds. I'll edit it further.